modules/search_api_solr_autocomplete/README.md

```diff
@@ -0,0 +1,25 @@
+Search API Solr Autocomplete
+-----------------------------
+
+ * Introduction
+ * Installation
+
+
+INTRODUCTION
+------------
+
+The `Search API Solr Autocomplete` module provides an integration between `Search API Solr` and `Search API Autocomplete`.
+
+ * For the description of the module visit:
+   https://www.drupal.org/project/search_api_solr_autocomplete
```

Comment:
Don't create a an extra project for a sub-module that is included in search_api_solr